### DISCUSSION TITLE
open Lapce from Command Line

### DISCUSSION THREAD

**Discussion by rodrigoromo28** [lapce/lapce#2400] [Q&A]
is there a command to use from command line to open Lapce, like gedit file.txt  for example

**Comment by panekj** [comment]
`lapce file.txt`

**Reply by barun-jha-maersk** [reply]
This is only working from lapce editor terminal not from system terminal

**Reply by geminixiang** [reply]
Command + P -> choose `:Install Lapce to PATH`, it will install at `/usr/local/bin/lapce`
